### PR TITLE
Suggestions width match search bar

### PR DESCRIPTION
**Pull request by eugenesk24**
### Description
The width of the suggestions dropdown match the width of the search bar. 

### Issues Resolved
When users had long queries the suggestions were far away from their cursor.

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [s] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by kavithacm — APPROVED** [review]
lgtm

File: dashboards-observability/public/components/common/search/autocomplete.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ import {
   AutocompleteState,
   createAutocomplete
 } from '@algolia/autocomplete-core';
-import { EuiTextArea } from '@elastic/eui';
+import { EuiFlexGroup, EuiFlexItem, EuiTextArea } from '@elastic/eui';
```
Comment (joshuali925) [inline dashboards-observability/public/components/common/search/autocomplete.tsx:23]: are these used?

**Review by joshuali925 — DISMISSED** [review]

File: dashboards-observability/public/components/common/search/autocomplete.tsx (hunk 1 above)
Comment (eugenesk24) [reply dashboards-observability/public/components/common/search/autocomplete.tsx:23]: No they're not, must have snuck in when I was testing them out.

**Review by joshuali925 — APPROVED** [review]

**Review by mengweieric — DISMISSED** [review]

**Review by ps48 — DISMISSED** [review]

File: dashboards-observability/public/components/common/search/search.scss [hunk 2]
```diff
@@ -25,18 +26,20 @@
...
-  max-width: unset;
-  height: 45px;
-  resize: vertical;
-  font-size: 16px;
-}
-
-.aa-Panel {
-  z-index: 2500;
+#autocomplete-root {
+  #autocomplete-textarea {
```
Comment (ps48) [inline dashboards-observability/public/components/common/search/search.scss:30]: Should root be parent of text-area?

**Review by mengweieric — APPROVED** [review]

**Review by mengweieric — APPROVED** [review]

**Review by ps48 — APPROVED** [review]

**Pull request merged**